include/substrait-mlir/Dialect/Substrait/IR/SubstraitOps.td

```diff
@@ -431,7 +431,7 @@ def Substrait_FetchOp : Substrait_RelOp<"fetch", [
   );
   let results = (outs Substrait_Relation:$result);
   let assemblyFormat = [{
-    $count (`offset` $offset^)? `from` $input attr-dict `:` type($input)
+    custom < CountAsAll > ($count) (`offset` $offset^)? `from` $input attr-dict `:` type($input)
```

Comment:
Nit: no spaces.
```suggestion
    custom<CountAsAll>($count) (`offset` $offset^)? `from` $input attr-dict `:` type($input)
```
Also: does this still fit in 80 characers?

Reply:
It does not fit in 80 characters. If I split into two lines that doesn't work with assembly format right?

Reply:
Yes, it does. It already uses the multi-line string format (`[{}]`) and line breaks in the assembly format are treated the same as any other whitespace, i.e., it results in one blank space character. So you can write:

```tablegen
    custom<CountAsAll>($count) (`offset` $offset^)? `from` $input
        attr-dict `:` type($input)
```